src/lib/src/ng-select.component.ts

```diff
@@ -521,11 +510,15 @@ export class NgSelectComponent implements OnInit, OnDestroy, OnChanges, ControlV
         }
     }
 
-    private isValueSet(value: any): boolean {
+    private get modelValue() {
```

Comment:
This name is a bit confusing because there is ngModel value which is not the same as this return value. I suggest to leave `get value(): NgOption  | NgOption[]` as is and add new getter `get selectedItems(): NgOption[]`  which will always return array so you can use selectedItems in HTML instead of value.